File: Project/src/app/models/airline.ts
import { Flight } from './flight';
import { Address } from './address';

export class Airline {
    id:number;
    name:string;
    address:Address;
    description:string;
    rate:number;
    flights:Array<Flight>;
    destinations:{};
    image:string;
    private city:string;

  constructor(id:number,name:string,address:Address,description:string,rate:number,
      flights:Array<Flight>,destinations:{},image:string){

    this.id = id;
    this.name=name;
    this.address=address;
    this.description = description;
    this.rate=rate;
    this.flights=flights;
    this.destinations=destinations;
    this.image = image;
    this.city=address.city;
  }


  getAddress():string{
    return this.address.toString();
  }
}

File: Server/Server/Models/Airline.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Models
{
    public class Airline
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [ForeignKey("OwnerId")]
        public string OwnerId { get; set; }

        public virtual RegisteredUser Owner { get; set; }
        public string Name { get; set; }

        [ForeignKey("AddressId")]
        public int AddressId { get; set; }

        public virtual Address Address { get; set; }

        public string Description { get; set; }

        public double Rate { get; set; }

        public string Logo { get; set; }

        public ICollection<Plane> Planes { get; set; }
        public ICollection<Destination> Destinations { get; set; } 
    }
}
